runbotics-orchestrator/src/main/java/com/runbotics/web/rest/GlobalVariableResource.java

```diff
@@ -85,7 +86,11 @@ public ResponseEntity<GlobalVariableDTO> createGlobalVariable(@RequestBody Globa
      * or with status {@code 500 (Internal Server Error)} if the globalVariableDTO couldn't be updated.
      * @throws URISyntaxException if the Location URI syntax is incorrect.
      */
-    @PreAuthorize("@securityService.checkFeatureKeyAccess('" + FeatureKeyConstants.GLOBAL_VARIABLE_EDIT + "')")
+    @PreAuthorize(
+        "@securityService.checkFeatureKeyAccess('" + FeatureKeyConstants.GLOBAL_VARIABLE_EDIT + "')" +
+        "and hasAuthority(\"" + AuthoritiesConstants.ADMIN + "\")" +
```

Comment:
aren't we assign featureKeys to Roles? I think we do, so there's only one out of two conditions needed

Reply:
imo keep the featureKey condition